doc/src/build/wallet.md

```diff
@@ -12,6 +12,33 @@ interface, *Wallet CLI*.
 
 Follow the instructions to [install Sui binaries](install.md#binaries).
 
+## Connect to DevNet
+We are hosting a public DevNet for the community to experiment with our 
+tech and help to shape the future of Sui network. To connect the wallet 
```

Comment:
Here we should probably inform the user to use the latest build with a link to [install Sui binaries](install.md#binaries) that shows how to upgrade the binaries. @Clay-Mysten is probably already working on it